tests/Response.test.ts

```diff
@@ -936,4 +1042,26 @@ describe('Response', () => {
 
    });
 
+   describe('_isValidJSONPCallback', () => {
+      let response: TestResponse;
+
+      beforeEach(() => {
+         response = new TestResponse(app, sampleReq, _.noop);
+      });
+
+      it('returns true for valid JSONP callbacks', () => {
+         expect(response._isValidJSONPCallback('callback')).to.strictlyEqual(true);
+         expect(response._isValidJSONPCallback('_abc')).to.strictlyEqual(true);
+         expect(response._isValidJSONPCallback('$')).to.strictlyEqual(true);
+         expect(response._isValidJSONPCallback('funcs[123]')).to.strictlyEqual(true);
```

Comment:
Can we add more tests? For example, I don't see anything with a `period`, and we should not only test single cases, but mixed cases too (i.e. `_` and `$` and `[..]` and combinations thereof).